Terminal.Gui/Views/TextView.cs

```diff
@@ -30,6 +30,8 @@
 using System.Text;
 using NStack;
 
+using Rune = System.Rune;
```

Comment:
This to fix a compiler error; disambiguate .NET5's System.Text.Rune with System.Rune coming from https://github.com/migueldeicaza/NStack/blob/447f61d/NStack/unicode/Rune.cs#L4.